packages/tools/pluggable-widgets-tools/src/typings-generator/index.ts

```diff
@@ -8,6 +8,7 @@ import { generateForWidget } from "./generate";
 const { mkdir, readFile, stat, writeFile } = promises;
 
 export async function transformPackage(content: string, basePath: string) {
+    const filePaths = [];
```

Comment:
I think we can get rid off this array as well and rename it to something like: `propsTypingFilePath`

Reply:
Let's keep the array. Could you rename the variable to something like: `propsTypingFilePaths`

Reply:
updated ✅